src/pages/checkout/[service_type]/[service_slug].jsx

```diff
@@ -175,14 +180,14 @@ function ServiceSlug() {
           style={{ flexShrink: 0, flexGrow: 1 }}
           overflow="auto"
         >
-          {!readyToSelectService && isThirdStep && serviceProps?.id && (
+          {!readyToSelectService && isSecondStep && serviceProps?.id && (
             <ServiceSummary service={serviceProps} />
           )}
           {readyToSelectService && (
             <SelectServicePlan />
           )}
-          {/* Fourth step */}
-          {!readyToSelectService && isFourthStep && (
+          {/* Third step */}
+          {!readyToSelectService && isThirdStep && (
```

Comment:
Handle these steps in a components where you will have the conditions and return the step that should be shown